app/views/anonymous_feedback/index.html.erb

```diff
@@ -11,7 +11,7 @@
                                scopes: @filtered_by) %>
     <%= hidden_field_tag :from, @dates.from if @dates.from.present? %>
     <%= hidden_field_tag :to, @dates.to if @dates.to.present? %>
-    <%= hidden_field_tag :paths, @filtered_by.paths if @filtered_by.paths.present? %>
+    <%= hidden_field_tag :paths, @filtered_by.path_id if @filtered_by.path_id.present? %>
```

Comment:
If the contents of this hidden field has changed, could the name change as well?

A field name like `path_id` might make more sense.

Reply:
Yes you're right, I will make that correction. Thanks

Reply:
Now I'm looking at this again, I think `path_id` is definitely better than `paths`, but something like `path_set_id` might be even clearer, as the id doesn't identify a single path, but a set of paths.

Reply:
I've changed `path_id` to `path_set_id`